File: problem_c.py
def problem_C():
    k = int(input())
    angles = [int(input()) for _ in range(k)]
    sum_of_angles = sum(angles)
    avrg = sum_of_angles // k
    if 360 % (180 - avrg) != 0:
        print(-1)
    else:
        min_n = 360 // (180 - avrg)
        max_n = avrg * k + 360 - 180 * k + k
        print(min_n, max_n)

problem_C()

File: problem_c.cpp
void problem_C(){
  int k;
  cin >> k;
  int a, sum=0;
  for(int i=0; i<k; i++){
    cin >> a;
    sum+=a;
  }
  int avrg = sum/k;
  if(360 % (180-avrg) != 0){  cout << -1;}
  else{
    int min_n = 360/(180-avrg);
    int max_n = (avrg*k + 360 - 180*k) + k;
    cout << min_n << " " << max_n << endl;
  }
}

int main(){
  problem_C();
  return 0;
}
